code/modules/clothing/clothing.dm

```diff
@@ -492,7 +492,7 @@
 	if(!no_slip)
 		return
 	var/mob/living/carbon/human/H = user
```

Comment:
Is it not better to do that check before it receives the user?

Reply:
Yeah just do an
```
if(!user)
      return
```
Above the typecast